### PR TITLE
Component groups list

### PR DESCRIPTION
**Pull request by tissakhosla**
Made a responsive groups list component for the members page.

### CODE REVIEW

**Comment by erinkelley27** [comment]
@tissakhosla Is there an issue for this with a mock? Or did you do the responsive look on your own?

**Comment by tissakhosla** [comment]
Hey I did make the responsive myself. Karen needed the component ASAP so I just tried to knock it out.

**Comment by tissakhosla** [comment]
Thank, lemme know if this is cool! :)

File: client/src/components/GroupsList/GroupsList.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.Groups__card--4 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 125px;
+  height: 150px;
+  text-align: center;
+}
+
+.Groups h1 {
```
Comment (erinkelley27) [inline client/src/components/GroupsList/GroupsList.css]: @tissakhosla Will you give this a className that is semantic and follows the BEM?

Comment (tissakhosla) [reply client/src/components/GroupsList/GroupsList.css]: NP... done :)

**Pull request merged**